packages/react/src/ActionList/Item.tsx

```diff
@@ -390,19 +396,18 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
                   </Box>
                   {slots.inlineDescription}
                 </ConditionalWrapper>
-                {
+                {slots.trailingVisual &&
```

Comment:
@tylerjdev it looks like this change is resulting in the inactive button missing in the snapshots - [Prod version](https://primer.style/react/storybook/?path=/story/components-actionlist-features--inactive-multiselect). 

(Side note: I am confused on if this button is here, just for the purpose of showing this info tooltip.. but that seems out-of-scope for now)

Reply:
I don't believe that this was an intentional change. We should be able to revert back to the prod state! I don't think this is needed for `TrailingAction`.